docs/pages/configuration/pipelines/functions/run_default_pipeline.mdx

```diff
@@ -2,3 +2,17 @@
 title: Function run_default_pipeline
 sidebar_label: run_default_pipeline
 ---
+
+Runs the default pipeline.
+
+## Arguments
+
+
+
+## Usage
+
+```
+run_default_pipeline [pipeline1]
```

Comment:
Consider removing the 1 after `pipeline1` since there's only ever one default pipeline.

Reply:
removed